module/Activity/src/Activity/Model/ActivityField.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Activity\Model;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Activity field model.
+ *
+ * @ORM\Entity
+ */
+class ActivityField
+{
+    /**
+     * ID for the field.
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", nullable=false)
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    protected $id;
+    
+    /**
+     * Activity that the field belongs to.
+     *
+     * @ORM\ManyToOne(targetEntity="Activity\Model\Activity", inversedBy="fields", cascade={"persist"})
+     * @ORM\JoinColumn(name="activity_id",referencedColumnName="id")
+     */
+    protected $activity;
+    
+    /**
+     * The name of the field.
+     * 
+     * @ORM\Column(type="string", nullable=false) 
+     */
+    protected $name;
+
+    /**
+     * The type of the field.
+     * 
+     * @ORM\Column(type="integer", nullable=false)
+     */
+    protected $type;
+    
+    /**
+     * The minimal value constraint for the ``number'' type
+     * 
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    protected $minimumValue;
+    
+    /**
+     * The maximal value constraint for the ``number'' type.
+     * 
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    protected $maximumValue;
+    
+    /**
+     * The allowed options for the field of the ``option'' type.
+     * 
+     * @ORM\OneToMany(targetEntity="ActivityOption", mappedBy="field")
+     */
+    protected $options;
+
+    /**
+     * Create a new field.
+     *
+     * @param array $params Parameters for the new field
+     * @param Activity $activity The activity 
+     *        the field is associated with
+     * @param EntityManager $em The relevant entity manager
+     * 
+     * @throws \Exception If a field is loaded
+     * @throws \Exception If a necessary parameter is not set
+     *
+     * @return \Activity\Model\ActivityField the created field
+     */   
+    public function create(array $params, Activity $activity, $em){
+        
+        if ($this->id != null) {
+            throw new \Exception('There is already a loaded activity');
+        }
+        
```

Comment:
Dit soort functies zouden eigenlijk beter in een service thuis horen. Dan is deze if statement bijvoorbeeld niet nodig. Alhoewel, deze manier van objecten aanmaken is wel consistent met hoe het in de rest van de activiteiten module gedaan wordt.